tutorials/notebooks/celestial-coords1/celestial_coords1.ipynb

```diff
@@ -0,0 +1,571 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ZjCbtC-bAtMr"
+   },
+   "source": [
+    "Let's print the `WCS` object one more time to check how our values were assigned."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 194
+    },
+    "colab_type": "code",
+    "id": "iwx8iECxJRs8",
+    "outputId": "d8a4818f-13b0-433f-d17d-f660e207ba4c"
+   },
+   "outputs": [],
+   "source": [
+    "print(wcs_helix_list) # To check output"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "rL7-Qut6AGDM"
+   },
+   "source": [
+    "Note that when we initialize the WCS object this way, the `NAXIS` values are set to 0. To assign coordinates to our image, we will need to fix the shape of the `WCS` object array so that it matches our image. We can do this by assigning a value to the `array_shape` attribute of the `WCS` object:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "6PnSCW0_-pQs"
+   },
+   "outputs": [],
+   "source": [
+    "wcs_helix_list.array_shape = [1024, 1024]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AszZGXxNBDoz"
+   },
+   "source": [
+    "Now when we print the `WCS` object, we can see that the `NAXIS` values have been updated from the default size of 0 to 1024."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 194
+    },
+    "colab_type": "code",
+    "id": "qdg6jLCa_Sck",
+    "outputId": "8285ba5c-10e8-457f-bded-15c3faff782c"
+   },
+   "outputs": [],
+   "source": [
+    "print(wcs_helix_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "qKvxquTdEOo5"
+   },
+   "source": [
+    "## Section 2: Show an image of the Helix nebula with RA and Dec labeled\n",
+    "\n",
+    "Most of the time we can obtain the required `astropy.wcs.WCS` object from the header of the FITS file from a telescope or astronomical database. This process is described below.\n",
+    "\n",
+    "### Step 1: Read in the FITS file\n",
+    "\n",
+    "First, we will download the FITS file from Zenodo using the Python requests form. Then we will read the FITS file into a variable called header_data_unit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "MRVbLwvcE__Y"
+   },
+   "outputs": [],
+   "source": [
+    "r = requests.get(\"https://zenodo.org/record/3946743/files/tailored_dss.22.29.38.50-20.50.13_60arcmin.fits\")\n",
+    "\n",
+    "with open('./tailored_dss.22.29.38.50-20.50.13_60arcmin.fits', 'wb') as f:\n",
```

Comment:
It would be more straightforward to do:
```
header_data_unit_list = fits.open('https://zenodo.org/record/3946743/files/tailored_dss.22.29.38.50-20.50.13_60arcmin.fits')
```

Reply:
Sure

Reply:
Updated with 
```python
header_data_unit_list = fits.open(...)
```